setup.py

```diff
@@ -11,14 +11,35 @@
     print("User must install the following packages manually:\n" +
           "\n".join(f' {r}' for r in git_requirements))
 
+# Widgets for Qt Designer via the pydm entrypoint
+designer_widgets = [
+    "TyphosAlarmCirclePlugin=typhos.alarm:TyphosAlarmCircle",
+    "TyphosAlarmEllipsePlugin=typhos.alarm:TyphosAlarmEllipse",
+    "TyphosAlarmPolygonPlugin=typhos.alarm:TyphosAlarmPolygon",
+    "TyphosAlarmRectanglePlugin=typhos.alarm:TyphosAlarmRectangle",
+    "TyphosAlarmTrianglePlugin=typhos.alarm:TyphosAlarmTriangle",
+    "TyphosCompositeSignalPanelPlugin=typhos.panel:TyphosCompositeSignalPanel",
+    "TyphosDeviceDisplayPlugin=typhos.display:TyphosDeviceDisplay",
+    "TyphosDisplaySwitcherPlugin=typhos.display:TyphosDisplaySwitcher",
+    "TyphosDisplayTitlePlugin=typhos.display:TyphosDisplayTitle",
+    "TyphosHelpFramePlugin=typhos.display:TyphosHelpFrame",
+    "TyphosMethodButtonPlugin=typhos.func:TyphosMethodButton",
+    "TyphosPositionerWidgetPlugin=typhos.positioner:TyphosPositionerWidget",
+    "TyphosRelatedSuiteButtonPlugin=typhos.related_display:TyphosRelatedSuiteButton",  # noqa
```

Comment:
Why did this line cause issues but none of the others?

Reply:
Just the long line length - I thought it would break the flow too much to break it up and opted for `noqa`, but would change if anyone felt strongly otherwise